staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/convert.go

```diff
@@ -80,7 +80,7 @@ func NewStructural(s *apiextensions.JSONSchemaProps) (*Structural, error) {
 		ss.Items = item
 	}
 
-	if len(s.Properties) > 0 {
+	if s.Properties != nil {
```

Comment:
this looks fine, but was this required for the defaulting validation to work properly?

Reply:
Was wondering: with proto, can you provide an empty, but defined properties field that ends up as empty map here?

Reply:
Background: we require to provide properties if non-preserve-unknown-fields embedded objects. The question behind ^^: is the user able to provide empty properties (i.e. enforce empty objects with only TypeMeta and ObjectMeta) ?